config/packages/test_cached/web_profiler.yaml

```diff
@@ -0,0 +1,6 @@
+web_profiler:
```

Comment:
Do not even ask why xD. @Zales0123 has found out that messenger is decorating with traceable message bus automatically only if ```data_collector.messenger``` is present.